docs/cristin.ttl

```diff
@@ -915,10 +926,12 @@ It is a self reference.""" ;
 :antall_sider_totalt
     a               :cristinfield ;
     rdfs:comment    """
-    The total number of pages.
+    The total number of pages of the specific entry.
+    When the the entry represents a part of another entry (e.g. a Chapter in a Book) then the total number of pages
+    refers to the entry (e.g Chapter) and not the to parent entry (e.g. Book).
```

Comment:
Looks like we're now using a bunch of terms for what one cristin result id represents: "result", "entry", "entity", "publication"...? Do we need to consolidate?

Reply:
I will create a separate PR for this